packages/uniforms-material/src/RadioField.tsx

```diff
@@ -46,7 +46,7 @@ function Radio({
       name={name}
       onChange={(event: any) => disabled || onChange(event.target.value)}
       ref={inputRef}
-      value={value}
+      value={value ?? 0}
```

Comment:
Ah! I meant `?? ''` instead of `|| ''` because of the `0`.
```suggestion
      value={value ?? ''}
```